gr-utils/python/modtool/cli/add.py

```diff
@@ -124,10 +124,17 @@ def get_copyrightholder(self):
         click.secho("For GNU Radio components the FSF is added as copyright holder",
                     fg='cyan')
 
+
 def get_arglist(self):
     """ Get the argument list of the block to be added """
-    if self.info['arglist'] is not None:
-        self.info['arglist'] = click.prompt(click.style('Enter valid argument list, including default arguments: \n', fg='cyan'), prompt_suffix='')
+    if self.info['arglist'] in [None,'']: 
```

Comment:
`if not self.info['arglist'] or self.info['arglist'].isspace()` will take care of spaces as well.